deep-learning/_posts/2018-07-05-Ship-wake-detection.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: review
+title:  "Ship wake-detection procedure using conjugate gradient trained artificial neural networks"
+tags:   deep-learning, remote-sensing, neural-network
+author: Charles Authier
+pdf:  https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=83986
+cite:
+  authors: "J.P. Fitch, S.K. Lehman, F.U. Dowla, S.Y. Lu, E.M. Johansson, D.M. Goodman"
+  title:   "Ship wake-detection procedure using conjugate gradient trained artificial neural networks"
+  venue:   " IEEE Transactions on Geoscience and Remote Sensing ( Volume: 29, Issue: 5, Sep 1991 )"
+---
+
+This paper developed a method to reduce large two-dimensional images to significantly smaller “feature lists.” These feature lists overcome the problem of storing and manipulating large amounts of data. A new artificial neural network using conjugate gradient training methods, operating on sets of feature lists, was successfully trained to determine the presence or absence of wakes in synthetic aperture radar images.
+
+**Images: radar, 4000 x 4000 pixels for a 25m of resolution by pixel**
+
+Because radar satellites refresh only a few seconds, the ocean-surface features such as internal wave signatures due to tidal flows over the continental shelf and ship wakes were observed in Seasat data to have a long duration and allow ship detection.
+
+In order to ensure that the entire wake appears in the digital representation, radar images would require between 10^6 and 10^8 pixels.
+
+## Main Idea
+
+A wake-detection system based on techniques for mapping input radar images into sets of one-dimensional feature lists, which can then be input to an artificial neural network. The mapping significantly reduces the quantity of data, which reduces the memory and training requirements so that an artificial neural network can be used.
+
+Results are presented for a wake-processing system operating on radar imagery produced by numerical simulation and by data collected with the NASA-JPL AirSAR. We show that a new conjugate gradient training.
```

Comment:
"We show that a new conjugate gradient training." ...? Missing something... :stuck_out_tongue: